cast/js/src/main/resources/prologue.js

```diff
@@ -109,6 +150,8 @@ Object.prototype = Object$proto$__WALA__;
 /* Function properties, see spec 15.3					*/
 /************************************************************************/
 
+//Function = function Function() {};
+
```

Comment:
Why this change?

Reply:
These were to identify the constructor calls. But, some of the tests fail as of now, due to these constructor calls. I can remove them for now.

Reply:
Remove the changes?  Or remove the tests?

Reply:
Remove the changes

Reply:
Were you planning to remove this change?

Reply:
done